### ISSUE TITLE
fatal: no path specified on deploy step

### ISSUE THREAD

**Issue by Briffou** [mhausenblas/mkdocs-deploy-gh-pages#8]
I think I missed something evident here, but I'm trying to use your deploy action, but get an error. The build failed in "Deploy docs" step:

```
Run mhausenblas/mkdocs-deploy-gh-pages@master
/usr/bin/docker run --name e87b52f2ef527d90a8492b9ca0f5d3d01dace3_2f656e --label e87b52 --workdir /github/workspace --rm -e PERSONAL_TOKEN -e HOME -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e GITHUB_ACTIONS=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/mkdocs_test/mkdocs_test":"/github/workspace" e87b52:f2ef527d90a8492b9ca0f5d3d01dace3
INFO    -  Cleaning site directory 
INFO    -  Building documentation to directory: /github/workspace/site 
WARNING -  Version check skipped: No version specificed in previous deployment. 
INFO    -  Copying '/github/workspace/site' to 'gh-pages' branch and pushing to GitHub. 
ERROR   -  Failed to deploy to GitHub with error: 
fatal: no path specified; see 'git help pull' for valid url syntax
```

You can see me config file here:
https://github.com/Briffou/mkdocs_test/blob/master/.github/workflows/build_mkdocs.yml

Do you have an hint ? (first time I use a "Github Actions")

**Comment by Briffou** [comment]
Ok, find my mistake, didn't know I have to had a "Secret" in **the repository settings** (and didn't understand this sentence in the doc :  "Generate a personal access token (repo) and add it to Secrets as PERSONAL_TOKEN.")

**Comment by mhausenblas** [comment]
Aha! Thanks for this. How would you rephrase it to make it more obvious? Can you send in a PR?
